tests/test_optimization_tf.py

```diff
@@ -76,12 +76,12 @@ def accumulate(grad1, grad2):
                 local_variables = strategy.experimental_local_results(gradient_placeholder)
                 local_variables[0].assign(grad1)
                 local_variables[1].assign(grad2)
-                strategy.experimental_run_v2(accumulate_on_replica, args=(gradient_placeholder,))
+                strategy.run(accumulate_on_replica, args=(gradient_placeholder,))
```

Comment:
I'm confused, if the strategy was just removed from experimental, how is this going to be compatible with versions <2.4?

Reply:
The `run` method is not experimental anymore since the TF 2.2 release, and removed of the experimental namespace since the 2.4.

Reply:
Ok but we are pinned to >=2.0

Reply:
Yep, at some point if we want to keep improve the TF codebase we have to change this sometime, this will be soon the case with the upcoming perf improvements. We will have to move to at least 2.3.

Reply:
As example here #9124

Reply:
I understand, but unless that moves happens today, we need some code that deals with both versions here.

Reply:
like this? (just pushed an update)